packages/rest/src/rest.server.ts

```diff
@@ -236,11 +237,20 @@ export class RestServer extends Context implements Server, HttpServerLike {
     });
 
     // Mount our error handler
-    this._expressApp.use(
-      (err: Error, req: Request, res: Response, next: Function) => {
-        this._onUnhandledError(req, res, err);
-      },
-    );
+    const errorHandler: ErrorRequestHandler = (err, req, res, next) => {
+      // Handle errors reported by Express middleware such as CORS
+      // First try to use the `REJECT` action
+      this.get(SequenceActions.REJECT, {optional: true})
```

Comment:
Can you please move this code to `_onUnhandledError` function (see below)? The function `_setupRequestHandlerIfNeeded` is intended to register different middleware, not to implement middleware handlers.